app/components/progress_tab/hypertension/control_component.rb

```diff
@@ -4,11 +4,10 @@ class ProgressTab::Hypertension::ControlComponent < ApplicationComponent
   include AssetsHelper
   include ActionView::Helpers::NumberHelper
 
-  attr_reader :control_range, :repository, :current_user
+  attr_reader :repository, :current_user
 
   def initialize(service, current_user)
-    @repository = service.control_rates_repository
-    @control_range = repository.range
+    @repository = service.repository
```

Comment:
I'm guessing that `service.repository` has the current month included, whereas `service.control_rates_repository.range` didnt?

Reply:
That's right. `service.repository` is initialized with the last 6 months data including the current month as the `periods` whereas `service.control_rates_repository`  is initialized with the last 12 months data excluding current month as `periods`